appveyor.yml

```diff
@@ -1,18 +1,34 @@
-version: 1.0.{build}
+version: 11.0.{build}
+pull_requests:
+  do_not_increment_build_number: true
 branches:
   only:
   - master
 image: Visual Studio 2017
+dotnet_csproj:
+  patch: true
+  file: src\ProgressOnderwijsUtils\ProgressOnderwijsUtils.csproj
+  version: '{version}'
+  package_version: '{version}'
+  assembly_version: '{version}'
+  file_version: '{version}'
+  informational_version: '{version}'
 environment:
   CONNECTION_STRING: Server=(local)\SQL2016;Database=master;User ID=sa;Password=Password12!
 services: mssql2016
 build_script:
 - ps: >-
-    dotnet build
+    dotnet build -c Release

```

Comment:
Is this required given that dotnet pack is called immediately hereafter?

Reply:
It is required if we want the tools project to be built as well.

Reply:
So this is both for deploy and testing purposes.